db/migrate/20160329231848_text_search_indices.rb

```diff
@@ -0,0 +1,12 @@
+class TextSearchIndices < ActiveRecord::Migration
+  def change
+    DbTextSearch::CaseInsensitiveEq.add_index connection, :thredded_categories, :name, name: :thredded_categories_name_ci
+    # Recreate previously existing MySQL index in favour of consistent names
+    if connection.adapter_name.downcase =~ /mysql/
+      remove_index :thredded_topics, :title
+      remove_index :thredded_posts, :content
+    end
+    DbTextSearch::FullTextSearch.add_index connection, :thredded_topics, :title, name: :thredded_topics_title_fts
+    DbTextSearch::FullTextSearch.add_index connection, :thredded_posts, :content, name: :thredded_posts_content_fts
```

Comment:
So just above, if it's mysql it's going to remove the `:thredded_topics` and `:thredded_posts` indexes. If it's pq does it not need to remove those indexes too?  Or are those JUST a mysql thing .. and don't exist with pg because pg is superior in every way?

Oops is my bias showing